### ISSUE TITLE
Any ways to paste from vim's internal register/clipboard?

### ISSUE THREAD

**Issue by qx-juilliard** [Yggdroot/LeaderF#909]
- vim or neovim? 
    - [x] vim
    - [ ] neovim
- Output of `vim --version` or `nvim --version`: 
- Output of `:echo has("python")`: 
- Output of `:echo has("python3")`: 
- Output of `:echo &pythondll`(only vim, not neovim): 
- Output of `:echo &pythonthreedll`(only vim, not neovim): 
- Output of `:py print(sys.version)`: 
- Output of `:py3 print(sys.version)`: 
- Output of `:echo g:Lf_Debug_Cmd`: 
- Output of `:echo g:Lf_FilesFromCache`: 
- Operating system: 
    - [x] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Etc.
- Configurations related to LeaderF in vimrc: 

### Describe your question, feature request, or bug.

I have the following use case: work on a file, then use, say `yy` to yank a line. Then I'd like to open LeaderF and paste what has just been `yy`ed to the "input".

In other words, is there anyway to map `<C-v>` (or other shortcuts) in input mode to custom vim commands? I found `g:Lf_CommandMap` but it doesn't seem to be relevant.

**Comment by Yggdroot** [comment]
ctrl-v can paste words from `+` register.

**Comment by qx-juilliard** [comment]
> ctrl-v can paste words from `+` register.

Thanks! Indeed I saw this option in the docs. However, my vim is not compiled with clipboard support. Is it possible to customize the behavior of `C-v` to paste from, say, the unnamed buffer (which would be convenient because all the `yy/dd` stuff works)?

**Comment by Yggdroot** [comment]
There is no configuration, you have to hack the code to achieve your goal.

**Comment by qx-juilliard** [comment]
> There is no configuration, you have to hack the code to achieve your goal.

OK I'll see if I can submit a patch to make it configurable
